101-cosmosdb-api-for-mongodb-account-create/azuredeploy.json

```diff
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"databaseAccountName": {
+			"type": "string",
+			"metadata": {
+				"description": "The MongoDB database account name."
```

Comment:
Hi @bmoore-msft, I'm not sure to get what you need here. Here are some definition around that:

- https://docs.microsoft.com/en-us/azure/templates/microsoft.documentdb/databaseaccounts
- https://github.com/Azure/azure-resource-manager-schemas/blob/master/schemas/2015-04-08/Microsoft.DocumentDB.json

Furthermore, when I'm looking some existing examples here: https://azure.microsoft.com/en-us/resources/templates/?resourceType=Microsoft.Documentdb, I'm not seeing any differences regarding the 'databaseAccountName' parameter. 

Feel free to let me know what do you need from me please. Thanks!